### ISSUE TITLE
Breaking change in HA 2024.5 beta

### ISSUE THREAD

**Issue by mwhdc** [jason0x43/hacs-hubitat#272]
Hubitat integration fails to set up entities in b0 and b1. An identical error is thrown for every domain that the Hubitat integration tries to populate (in my case, alarm_control_panel, cover, light, lock, switch, fan).

```
2024-04-27 11:38:55.271 ERROR (MainThread) [homeassistant.components.alarm_control_panel] Error while setting up hubitat platform for alarm_control_panel
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 356, in _async_setup_platform
    await asyncio.shield(awaitable)
  File "/config/custom_components/hubitat/alarm_control_panel.py", line 195, in async_setup_entry
    create_and_add_entities(
  File "/config/custom_components/hubitat/entities.py", line 29, in create_and_add_entities
    hub = get_hub(hass, config_entry.entry_id)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hubitat/hub.py", line 503, in get_hub
    return hass.data[DOMAIN][config_entry_id]
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
KeyError: '[32 hex digits]'
```

**Comment by jason0x43** [comment]
This should be fixed in v0.9.28.

**Comment by mwhdc** [comment]
Working well with 2024.5.0b1. Thanks for the quick fix.
